pulp_container/app/json_schemas.py

```diff
@@ -1,19 +1,22 @@
 from pulp_container.constants import (
     ALLOWED_ARTIFACT_TYPES,
-    ALLOWED_BLOB_CONTENT_TYPES,
     BLOB_CONTENT_TYPE,
     MEDIA_TYPE,
     SIGNATURE_TYPE,
 )
 
 
 def get_descriptor_schema(
-    allowed_media_types, additional_properties=None, additional_required=None
+    allowed_media_types=[], additional_properties=None, additional_required=None
 ):
     """Return a concrete descriptor schema for manifests."""
 
+    MEDIA_TYPE_CONFIG = {"type": "string"}
```

Comment:
Please, try using variables with lowercase inside the function's scope. This is not a real constant, is it?

Besides that, can you point me to the specification that allows an *empty* media type? The `get_descriptor_schema` function is used in various places, not just for configs.